_2020/data-wrangling.md

```diff
@@ -220,7 +225,7 @@ ssh myserver journalctl
 `sort` will, well, sort its input. `uniq -c` will collapse consecutive
 lines that are the same into a single line, prefixed with a count of the
 number of occurrences. We probably want to sort that too and only keep
-the most common logins:
+the most common usernames:
```

Comment:
I'd like this to remain "logins".

Reply:
Totally your call. You referred to them as "usernames" a few lines prior, and not all of those usernames succeeded in logging in, so I thought that "usernames" might be more appropriate and consistent than "logins".

Reply:
Ah, yes, good point!